api/pkg/apis/v1alpha1/managers/activations/activations-manager.go

```diff
@@ -389,3 +421,36 @@ func (t *ActivationsManager) ValidateCreateOrUpdate(ctx context.Context, state m
 func (t *ActivationsManager) CampaignLookup(ctx context.Context, name string, namespace string) (interface{}, error) {
 	return states.GetObjectState(ctx, t.StateProvider, validation.Campaign, name, namespace)
 }
+
+func getOutputSizeOf(obj interface{}) int {
```

Comment:
Just curious = would measuring JSON serialized bytes faster?

Reply:
Yes you are right. Let me reconsider how to truncate with JSON serialized bytes length.